realm/realm-library/src/main/java/io/realm/RealmList.java

```diff
@@ -108,10 +108,10 @@ public RealmList(E... objects) {
     }
 
     /**
-     * Checks if {@link io.realm.RealmResults} is still valid to use i.e. the {@link io.realm.Realm} instance hasn't
-     * been closed.
+     * Checks if the {@link RealmList} is managed by Realm and contains valid data i.e. the {@link io.realm.Realm}
+     * instance hasn't been closed.
```

Comment:
'the {@link io.realm.Realm} instance hasn't been closed.`

should be 

`the {@link io.realm.Realm} instance hasn't been closed and the container Realm object has not been deleted.`

?